common/src/primitives/compact.rs

```diff
@@ -20,6 +20,17 @@ use std::ops::Shl;
 #[derive(PartialEq, Eq, PartialOrd, Ord, Hash, Clone, Copy, Encode, Decode)]
 pub struct Compact(pub u32);
 
+impl Compact {
+    pub fn highest_value() -> Self {
+        Uint256([0xFFFFFFFFFFFFFFFF, 0xFFFFFFFFFFFFFFFF, 0xFFFFFFFFFFFFFFFF, 0xFFFFFFFFFFFFFFFF])
+            .into()
+    }
+
+    pub fn lowest_value() -> Self {
+        Uint256([0, 0, 0, 0]).into()
```

Comment:
`Uint256::ZERO`

Reply:
Also, if the conversion from `Uint256` to `Compact` was made a `const fn` (which should be possible), the `{lowest,highest}_value` could be constants.

Reply:
Ah nice.

Reply:
Same here.